mmcls/datasets/cub.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+
+from .base_dataset import BaseDataset
+from .builder import DATASETS
+
+
+@DATASETS.register_module()
+class CUB(BaseDataset):
+    """`CUB-200-2011.
```

Comment:
please modify this line refer to https://github.com/open-mmlab/mmclassification/blob/a7f8e96b31c10ab3e9c133293ca406e6e548475b/mmcls/datasets/cifar.py#L17

Reply:
I'm getting this problem with docformatter's auto-formatting. What should I do?